cfme/common/provider.py

```diff
@@ -92,6 +92,13 @@ def __init__(self, **kwargs):
                 self.token = kwargs['token']
 
     @property
+    def properties_form(self):
```

Comment:
Well, my idea was to try to make it uniform across all providers, if you look at infrastructure, all that is required is setting it to ```_properties_form``` instead of ```properties_form```. I don't see a big issue here, if I go through and make the change in middleware and containers too. Thoughts?